### DISCUSSION TITLE
Customize database notifications icons

### DISCUSSION THREAD

**Discussion by binaryfire** [filamentphp/filament#9335] [Ideas]
Hey @zepfietje.

I'm interested in working on a PR to allow customization of the db notifications trigger icon, empty icon and text. I've got an app where I want to use db notifications solely for feature announcements. Apps that do this usually use a loudspeaker icon instead of a bell:

<img width="200" alt="image" src="https://github.com/filamentphp/filament/assets/5361908/3af8b151-909c-421d-9513-d4361699eb83">

A bell would be misleading because users would be expecting notifications related to their activities on the platform. Clicking a bell and seeing product announcements would feel a bit spammy.

What do you think? And what would be the best way to implement it? I see the icons are hardcoded [here](https://github.com/filamentphp/filament/blob/bb67bb5201eddefb7d76ff65668deabaf9b81cbe/packages/panels/resources/views/components/topbar/database-notifications-trigger.blade.php#L4) and [here](https://github.com/filamentphp/filament/blob/bb67bb5201eddefb7d76ff65668deabaf9b81cbe/packages/notifications/resources/views/components/database/modal/index.blade.php#L18). What about a config file option?

**Comment by zepfietje** [comment]
Have a look at the docs. 😉
https://filamentphp.com/docs/3.x/support/icons#available-icon-aliases

Text can be tweaked using translations.
If I'm missing something though, let me know.

**Comment by binaryfire** [comment]
@zepfietje Nice! Had no idea that was possible.
